nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c

```diff
@@ -309,7 +309,7 @@ Reply   Total	Cached	Bits In			Bits Out		Bits/Reply	  Ratio
   if (!(dispatchException & DE_TERMINATE))
     dispatchException = 0;
 
-    while (!dispatchException)
+  while (!dispatchException)
```

Comment:
In theory you'd have to remove the wrong indentation level for the whole block... which is a bit tricky in the first place since it mixes tabs and spaces...

Reply:
> In theory you'd have to remove the wrong indentation level for the whole block... which is a bit tricky in the first place since it mixes tabs and spaces...

only in theory... (!)